scripts/githooks/editorconfig-pre-commit.sh

```diff
@@ -1,32 +1,79 @@
 #!/bin/bash
 
-# Pre-commit git hook to check the EditorConfig rules complience over changed files
+set +e
```

Comment:
Can I ask what this does?

Reply:
The `set +e` option instructs the shell interpreter to exit immediately if an exit code of a command is different than `0` (`0` exit code means a successful execution).

The reason why we don't put it as `#!/bin/bash +e` is to make it more explicit as it affects the code behaviour. In the other script, we have chosen to go with `set -e` as it requires for an error to be `caught` for the output to be printed to the console and then execution resumed.